### PR TITLE
chore(suite): sentry tree shaking

### LINKED ISSUE #6444: Sentry Tree Shaking
configure sentry tree shaking to save some bundle size:
https://docs.sentry.io/platforms/javascript/guides/electron/configuration/tree-shaking/

### PR DESCRIPTION
**Pull request by AdamSchinzel**
Minimized the bundle size of the Sentry SDK

## Description

Set 2 flags in the Sentry SDK to `false`:
1. `__SENTRY_DEBUG__` - removes code related to debug logging
2. `__SENTRY_TRACING__` - removes code related to performance monitoring (needs to be removed when this is introduced in the app)

## Related Issue

Resolve #6444

### CODE REVIEW

File: packages/suite-build/configs/base.webpack.config.ts [hunk 1]
```diff
@@ -172,6 +172,8 @@ const config: webpack.Configuration = {
             'process.env.JWS_PUBLIC_KEY': JSON.stringify(jwsPublicKey),
             'process.env.CODESIGN_BUILD': isCodesignBuild,
             'process.env.SENTRY_RELEASE': JSON.stringify(sentryRelease),
+            _SENTRY_DEBUG__: false,
```
Comment (tomasklim) [inline packages/suite-build/configs/base.webpack.config.ts]:
```suggestion
            __SENTRY_DEBUG__: false,
```

**Review by tomasklim — CHANGES_REQUESTED** [review]

**Comment by tomasklim** [comment]
Looks like it does something
Before:
Screenshot 2023-08-01 at 9 09 50
After:
Screenshot 2023-08-01 at 9 15 16

**Comment by AdamSchinzel** [comment]
> Looks like it does something Before: Screenshot 2023-08-01 at 9 09 50 After: Screenshot 2023-08-01 at 9 15 16

First is 313.361 KB and second is 282.273 KB so we dropped approximately 31.088 KB. Not bad

**Comment by matejkriz** [comment]
Great, but please, set it just for "production" build not dev build, Sentry debug mode is useful on localhost for development.

**Comment by AdamSchinzel** [comment]
> Great, but please, set it just for "production" build not dev build, Sentry debug mode is useful on localhost for development.

Done

File: packages/suite-build/configs/base.webpack.config.ts [hunk 2]
```diff
@@ -163,16 +163,31 @@ const config: webpack.Configuration = {
...
+                      'process.env.VERSION': JSON.stringify(suiteVersion),
+                      'process.env.COMMITHASH': JSON.stringify(gitRevision),
+                      'process.env.ASSET_PREFIX': JSON.stringify(assetPrefix),
+                      'process.env.JWS_PUBLIC_KEY': JSON.stringify(jwsPublicKey),
+                      'process.env.CODESIGN_BUILD': isCodesignBuild,
+                      'process.env.SENTRY_RELEASE': JSON.stringify(sentryRelease),
+                      __SENTRY_DEBUG__: false,
+                      __SENTRY_TRACING__: false, // needs to be removed when we introduce performance monitoring in trezor-suite
+                  },
+        ),
```
Comment (matejkriz) [inline packages/suite-build/configs/base.webpack.config.ts]: I don't like the duplication, what about something like
```suggestion
         new webpack.DefinePlugin({
             'process.browser': true,
             'process.env.SUITE_TYPE': JSON.stringify(project),
             'process.env.VERSION': JSON.stringify(suiteVersion),
             'process.env.COMMITHASH': JSON.stringify(gitRevision),
             'process.env.ASSET_PREFIX': JSON.stringify(assetPrefix),
             'process.env.JWS_PUBLIC_KEY': JSON.stringify(jwsPublicKey),
             'process.env.CODESIGN_BUILD': isCodesignBuild,
             'process.env.SENTRY_RELEASE': JSON.stringify(sentryRelease),
             ...(isDev ? { 
                __SENTRY_DEBUG__: false,
                __SENTRY_TRACING__: false, // needs to be removed when we introduce performance monitoring in
             } : {}
         }),
```

Comment (AdamSchinzel) [reply packages/suite-build/configs/base.webpack.config.ts]: How about how it is now?

**Review by matejkriz — APPROVED** [review]
👍 nice, works fine, even though `__SENTRY_DEBUG__` is replaced by `true` in dev, it does not turn on debug mode by default, it can be turned on and off via sentry config same as before 👌

**Review by tomasklim — APPROVED** [review]

**Pull request merged**